### ISSUE TITLE
HttpWriter can't acces the web.xml mimetype.

### ISSUE THREAD

**Issue by MorganMaison** [BriteSnow/snow#48]
HttpWriter can never read the mimetype defined by the http server and is using the internal extramimetypes.properties instead.

**Comment by jeremychone** [comment]
Not sure, it does use the servlet contect getMimeType, no? 

```java
    public String getContentType(String resourcePath) {
        String contentType = null;
        if (servletContext != null) {
            servletContext.getMimeType(resourcePath);
        }
        // if the servletContext (server) could not fine the mimeType, then, give a little help
        if (contentType == null) {
            contentType = FileUtil.getExtraMimeType(resourcePath);
        }
        return contentType;
    }
```

**Comment by jeremychone** [comment]
Arrrr, sorry, silly bug. Forget the ```contentType = ....``` in the first if.

**Comment by jeremychone** [comment]
Ok, this was fixed, please test 2.0.5-SNAPSHOT and let me know if it is fixed.

**Comment by MorganMaison** [comment]
2.0.5-SNAPSHOT tested and yes it is fixed.
